### PR TITLE
feat(datasource): Add perfomance tuning for bonita datasource

### PR DESCRIPTION
**Pull request by danila-m**
Relates to [RUNTIME-474](https://bonitasoft.atlassian.net/browse/RUNTIME-474)

### CODE REVIEW

**Comment by baptistemesta** [comment]
Do we have a troubleshooting section somewhere. 
I guess the goal was to explain that if the platform admin encounters connection issue like it happened in the customer case,  he should apply that fix.
If you think it's not usefull to add it there because it's already in the product, it';s better to not add anything more in the doc on that.

File: modules/ROOT/pages/performance-tuning.adoc [hunk 1]
```diff
@@ -281,6 +281,20 @@ Two datasources are defined:
 
 Note that the sum of the maximum values configured for bonitaDS and bonitaSequenceManagerDS should be less than or equal to the maximum number of simultaneous connections allowed to your database.
 
+TIP: While the configuration of the 2 provided datasources works,
```
Comment (alachambre) [inline modules/ROOT/pages/performance-tuning.adoc:284]: You TIP doesn't embed the code example, is it what you had in mind?

**Comment by alachambre** [comment]
@danila-m don't forget your pull request please :)

File: modules/ROOT/pages/performance-tuning.adoc (hunk 1 above)
Comment (educhastenier) [reply modules/ROOT/pages/performance-tuning.adoc:284]: Up @danila-m

**Pull request closed without merging**